### PR TITLE
Properly detect cluster chain end markers & Update Unit Test instructions

### PR DESCRIPTION
**Pull request by johnwayner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,25 +148,30 @@ You can then apply the enforced style by typing:
...
-make gtest/bin/mega65_ftp.test
-```
-
-To run the unit tests:
+To generate the tests and run them:
 
 ```
-cd gtest/bin
-./mega65_ftp.test
+make test
```
Comment (gurcei) [inline README.md:176]: Maybe mention both linux and windows methods:

For linux: `make test`
For windows: `make test.exe`

**Review by gurcei — CHANGES_REQUESTED** [review]
Looking good, just one suggestion to mention the windows way to run the tests too :)

File: README.md (hunk 1 above)
Comment (johnwayner) [reply README.md:177]: Updated.

**Review by gurcei — APPROVED** [review]
All good, thanks for your efforts man! :D

I'll do a git squash-merge now and we're good :)

**Pull request merged**